SU-SWS/stanford_paragraph_types
# EARTH-89: Added section header to viewfield paragraph type.

# READY FOR REVIEW

# Summary
- Adds a section header to the view field so that the content author can combine 

# Needed By (Date)
- End of sprint (24th)

# Steps to Test

1. Check out this branch and EARTH-89 of stanford_research_area
2. Run `drush fia -y`
3. Clear caches and edit the home page node
4. Add a section heading to the viewfield paragraph at the bottom
5. Review front end and styling

Right now the the header stays above the view and we can work with Joe later to get a beside option as well.

# Affected Projects or Products
- EARTH-89
- Research challenge areas view

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
